README.rst

```diff
@@ -1,14 +1,16 @@
 Run-Your-Own Firefox Sync Server
 ================================
 
+Note: For the duration of Q4 2020 this documentation will be revised. This situation will only be temporary until tokenserver is ported to rust and this documentation is rewritten to leave out the old dependencies.
```

Comment:
May want to emphasize this a bit. maybe 

`**Note:**`
or offset things in a block quote just to make sure folks see that.